godtools/Views/TractElements/TractPage.swift

```diff
@@ -15,9 +15,15 @@ import SWXMLHash
 
 class TractPage: BaseTractElement {
     
+    var pageContainer: TractPageContainer?
+    
     // MARK: - Setup
     static var navbarHeight: CGFloat = 0.0
     
+    static var statusbarHeight: CGFloat {
+        return UIDevice.current.iPhoneX() ? CGFloat(44) : CGFloat(0)
```

Comment:
These should probably use the constants I created in `TractViewController` . The would have to be made static though.